pkg/apis/core/types.go

```diff
@@ -698,6 +707,15 @@ type ISCSIPersistentVolumeSource struct {
 	// <target portal>:<volume name> will be created for the connection.
 	// +optional
 	InitiatorName *string
+	// ReplacementTimeout controls how long the iSCSI layer should wait for a
+	// timed-out path/session to reestablish itself before failing any commands on it.
+	// If the ReplacementTimeout value is 0, IO will be failed immediately.
+	// If the ReplacementTImeout value is less than 0, IO will remain queued until the session
```

Comment:
https://github.com/open-iscsi/open-iscsi does not describe meaning of negative value. How do you know that negative value means basically no timeout?

Reply:
@jsafrane https://github.com/open-iscsi/open-iscsi/blob/master/etc/iscsid.conf in Timeout section described about the negative value of replacement timeout.